packages/cozy-doctypes/src/banking/Makefile

```diff
@@ -0,0 +1,5 @@
+encrypted.tar.gz.gpg: __snapshots__/matching.spec.js.snap fixtures
```

Comment:
I prefer to have this command directly on `package.json` it's more easy to find all command on the same file

Reply:
After explanation with @ptbrowne it's a good idea. I did not know the usefulness of makefile to handle dependencies 👍